From 2ed8a924be1dee3a1853fc17ac9062cf4dd5a0fc Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Sat, 17 Feb 2018 19:39:38 +0100
Subject: [PATCH] Allow deletion of symlink

Change-Id: I3c113f019fbf1264a8a54b4b9982df8a73abe91d
---

diff --git a/Ext4Crypt.cpp b/Ext4Crypt.cpp
index dc2e42a..6398464 100644
--- a/Ext4Crypt.cpp
+++ b/Ext4Crypt.cpp
@@ -211,6 +211,7 @@
 static bool destroy_dir(const std::string& dir) {
     LOG(DEBUG) << "Destroying: " << dir;
     if (rmdir(dir.c_str()) != 0 && errno != ENOENT) {
+        if(unlink(dir.c_str()) == 0) return true;
         PLOG(ERROR) << "Failed to destroy " << dir;
         return false;
     }
